RadarSDK/RadarUtils.m

```diff
@@ -64,6 +66,69 @@ + (NSString *)deviceMake {
     return @"Apple";
 }
 
++ (RadarConnectionType)networkType {
+    SCNetworkReachabilityRef reachability = SCNetworkReachabilityCreateWithName(kCFAllocatorDefault, "google.com");
+    if(!reachability) {
+        return RadarConnectionTypeUnknown;
+    }
+    
+    SCNetworkReachabilityFlags flags;
+    if(!SCNetworkReachabilityGetFlags(reachability, &flags)) {
+        CFRelease(reachability);
+        return RadarConnectionTypeUnknown;
+    }
+    
+    CFRelease(reachability);
+    
+    BOOL isReachable = (flags & kSCNetworkReachabilityFlagsReachable) != 0;
+    BOOL isWWAN = (flags & kSCNetworkReachabilityFlagsIsWWAN) != 0;
+    
+    if (isReachable) {
+        if (isWWAN) {
+            CTTelephonyNetworkInfo *networkInfo = [[CTTelephonyNetworkInfo alloc] init];
+            
+            NSDictionary *carrierTypes = networkInfo.serviceCurrentRadioAccessTechnology;
+            if (carrierTypes && carrierTypes.count > 0) {
+                return RadarConnectionTypeCellular;
```

Comment:
What else is on this `networkInfo` object? I imagine there might be more here we'd want.

In the field last week, we were asking people for their carrier... I'd almost imagine wanting to send that as well tbh.

Reply:
I could imagine listening to this delegate and calling RadarLogger each time there's a change (behind debug?): https://developer.apple.com/documentation/coretelephony/cttelephonynetworkinfo/delegate

Reply:
This is interesting, I think worth including: https://developer.apple.com/documentation/coretelephony/ctcellulardata